### PR TITLE
[FEAT] Nested navigation

### PR DESCRIPTION
**Pull request by pzuraq**
Adds styles and makes some changes in preparation for nested navigation

Screen Shot 2019-09-24 at 1 33 04 AM

Depends on https://github.com/empress/guidemaker/pull/21

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "ember-prism": "^0.4.0",
     "ember-styleguide": "^2.5.0",
     "ember-tether": "^1.0.0",
+    "ember-truth-helpers": "^2.1.0",
```
Comment (rwjblue) [inline package.json]: Hmm, it looks like we were already using `eq` (without having this dependency), can you double check that the app itself doesn't add a manual `eq`?

**Comment by rwjblue** [comment]
BTW, are there tests for the navigation bits? These combos seem pretty useful to test IMHO...

**Comment by rwjblue** [comment]
Looks like there are no tests for `chapter-links` component (tracking that in https://github.com/ember-learn/guidemaker-ember-template/issues/26), could you maybe add one or two for the new functionality added here?

File: package.json (hunk 1 above)
Comment (pzuraq) [reply package.json]: It doesn't, I believe this was a dependency of another addon actually, it was in the lockfile

**Comment by pzuraq** [comment]
failures appear to only be on Ember v2, looking into them but do we want to keep supporting versions of Ember that old with this template? I get supporting v2.18 in Guidemaker proper (unsure if anyone is actually using it besides us though) but it feels like it may just be debt to support such old versions in our own template.

**Comment by pzuraq** [comment]
Yeah, actually the failures here aren't even caused by my code, they're caused by Guidemaker, specifically this line: https://github.com/empress/guidemaker/blob/master/addon/services/head-data.js#L12

It looks like the HeadData service is only compatible with Ember v3+, and the only reason tests are passing at the moment on older versions of Ember is because we don't actually thoroughly test things. I'm going to drop support for all v2 versions from this lib, we should consider dropping support for them in Guidemaker too given this.

**Pull request merged**